ticketbuyer/buyer.go

```diff
@@ -323,13 +301,23 @@ func (t *TicketPurchaser) Purchase(height int64) (*PurchaseStats, error) {
 	}
 	ps.PriceNext = sDiffEsts.Expected
 
-	// Scale the average price according to the configuration parameters
-	// to find minimum and maximum prices for users that are electing to
-	// attempting to manipulate the stake difficulty.
 	maxPriceAbsAmt, err := dcrutil.NewAmount(t.cfg.MaxPriceAbsolute)
 	if err != nil {
 		return ps, err
 	}
+	// a sensible default, 25% above avgprice
+	negone, err := dcrutil.NewAmount(-1)
+	if maxPriceAbsAmt == negone {
+		maxPriceAbsAmt, err = dcrutil.NewAmount(avgPrice * 1.25)
```

Comment:
Please make the 1.25 into a default setting instead of using a magic number here.